include/delivery_reservation_lamp_manager/delivery_reservation_lamp_manager.hpp

```diff
@@ -27,28 +28,61 @@ class DeliveryReservationLampManager : public rclcpp::Node
   explicit DeliveryReservationLampManager(const rclcpp::NodeOptions & options);
   ~DeliveryReservationLampManager();
 
+  enum BlinkType
+  {
+    FAST,
+    SLOW,
```

Comment:
To match TWO_BLINKS, it would be FAST_BLINK and SLOW_BLINK.

Reply:
The name of the definition was changed.